docs/application/dotnet/guides/overview.md

```diff
@@ -49,9 +49,9 @@ You can use the following TizenFX features in your .NET applications:
 
```

Comment:
Line 4:
Global comment: 
Use Tizen.Wearable.CircularUI with inline code-block in MD file as ‘Tizen.Wearable.CircularUI’
Drop hyphen from “wearable-specific" and rephrase it to “wearable specific”

Line 33:
Kindly rephrase “…a 2-dimensional or a 3-dimensional…” to “a 2D or a 3D” (Source: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/numbers-symbols/3d-three-dimensional) 

Line 37:
Please remove anchor tag for hyperlink. Kindly insert hyperlink as per the MD standards as [Xamarin.Forms](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/). For more information kindly check how to insert link in MD file from: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
I'm not in charge of those statements so can't modify those statements.

Reply:
@hsgwon OK. Please incorporate all other changes recommended, where you have access.